web-app/src/components/BookingModal/index.js

```diff
@@ -1,102 +1,118 @@
 import React from 'react';
-import DatePicker from 'react-datepicker';
 import { PropTypes as PT } from 'prop-types';
-import { Button, Modal } from '../common';
+import { Button, Modal, Input } from '../common';
 import FontAwesomeIcon from '@fortawesome/react-fontawesome';
 import { faTimes } from '@fortawesome/fontawesome-free-solid';
-import { StyleContainer, BookingStatus, BookingInputs, StatusDot, ButtonWrap } from './styled';
+import {
+  StyleContainer,
+  BookingStatus,
+  StatusDot,
+  ButtonWrap,
+  Form,
+} from './styled';
 import { statusText } from '../../utilities/holidayStatus';
 
 const BookingModal = props => {
   const {
     closeModal,
     showModal,
     booking,
-    changeStart,
-    changeEnd,
-    changeHalfday,
+    formElementsArray,
     requestHoliday,
     cancelHoliday,
     updateHoliday,
-    changeWFH,
+    formChanged,
   } = props;
 
-  const getActions = () => (
-    booking.title ?
+  const getActions = () =>
+    booking.title ? (
       <ButtonWrap>
-        <Button id="updateHolidayBtn" onClick={updateHoliday} label={'Update'} />
-        <Button id="cancelHolidayBtn" onClick={cancelHoliday} label={'Cancel Booking'} />
+        <Button
```

Comment:
Quite a big ternary here. This could be broken down a bit more with logic inside the button component.

Reply:
Yeah, I totally agree. I'll update it now. thanks